webapps/api/backend/views/tasks.py

```diff
@@ -59,15 +59,16 @@ def retrieve(self, pk):
         :param pk: Primary key
         """
         result = celery.AsyncResult(pk)
+        status = result.status
```

Comment:
You cannot name your variable ```status``` here, as this shadows the rest_framework status module, which will cause some error that ```status``` has no property ```HTTP_200_OK```.